astropy/wcs/wcsapi/high_level_api.py

```diff
@@ -40,12 +40,13 @@ def low_level_wcs(self):
     @abc.abstractmethod
     def pixel_to_world(self, *pixel_arrays):
         """
-        Convert pixel coordinates to world coordinates (represented by high-level
-        objects).
+        Convert pixel coordinates to world coordinates (represented by
+        high-level objects).
 
-        If a single high-level object is used to represent the world
-        coordinates, it is returned as-is (not in a tuple/list), otherwise a
-        tuple of high-level objects is returned. See
+        If a single high-level object is used to represent the world coordinates
+        (i.e.. if ``len(wcs.world_axis_object_classes) == 1``), it is returned
```

Comment:
``i.e..`` -> ``i.e.`` or ``i.e.,`` (I prefer the former)

Reply:
As I recall the style guide prefers the latter. It's not merged yet though, but it's clear that it would be great to have some basic flake8 like tool to check on a few things and spelling, etc otherwise we'll diverge super quickly.

Reply:
I think ApJ used to make me write `i.e.,` with a comma.